hello-world/AKS/helloworld.yaml

```diff
@@ -9,7 +9,7 @@ spec:
   containers:
     - image: "fishersnpregistry.azurecr.io/hello-world-aks:1.1"
```

Comment:
We need to get this updated to a new registry.

Reply:
Updated the images and the locations to the official mcr